ckanext/ckanext-apply_permissions_for_service/ckanext/apply_permissions_for_service/logic/action.py

```diff
@@ -50,8 +50,8 @@ def service_permission_application_create(context, data_dict):
     if service_code_list is None or service_code_list == "":
         errors['service_code_list'] = _('Missing value')
 
-    intermediate_organization_id = data_dict.get('intermediate_organization_id')
-    intermediate_business_code = data_dict.get('intermediate_business_code')
+    intermediate_organization_id = data_dict.get('intermediate_organization_id') or None
+    intermediate_business_code = data_dict.get('intermediate_business_code') or None
```

Comment:
.get() returns None if the key is not found in the dict so this has no effect.

Reply:
Fixed